test/org/openlcb/GatewayTest.java

```diff
@@ -20,8 +19,8 @@ public void testCtor() {
     @Test    
     public void testGet() {
```

Comment:
This test needs to have some assertions added to it.  You removed setting of two unused variables, but what really should happen is that we should assert that some value is provided for g, g.getEastConnection and g.getWestConnection

Reply:
Added assertions